.github/workflows/tests.yml

```diff
@@ -90,30 +117,51 @@ jobs:
         env:
           # https://github.com/shivammathur/setup-php/issues/407#issuecomment-773675741
           fail-fast: true
-
+      - name: Check phpinfo()
+        run: php -i
+      - name: Check Composer Version
+        run: composer -V
+      - name: Check PHP Extensions
+        run: php -m
       # https://github.com/zf1s/zf1/pull/6#issuecomment-495397170
-      - name: Validate composer.json for all packages
+      - name: Validate composer.json For All Packages
         run: |
           for json in composer.json packages/*/composer.json; do echo COMPOSER=$json; done | parallel env {} composer validate
 
-      - name: Install dependencies
+      - name: Install Dependencies
         run: composer install --prefer-dist --no-progress --no-suggest
 
-      - name: Lint PHP source files
+      - name: Lint PHP Source Files
         run: |
           composer lint.checkstyle | cs2pr
 
-      - name: Setup environment for PHPUnit
+      - name: Setup Environment For PHPUnit
         run: |
           cp tests/TestConfiguration.ci.php tests/TestConfiguration.php
           echo "Existing locales"
           locale -a
           sudo apt-get update && sudo apt-get install tzdata locales -y && sudo locale-gen $LOCALES
           echo "All languages..."
           locale -a
+          echo "Installing ldap client"
+          sudo apt-get install -y libnss-ldap libpam-ldap ldap-utils
+
+      - name: Import LDAP Configuration
+        run: |
+          echo -e "Applying ACL mod for zf1..."
```

Comment:
is this block in yaml needed? you can just bind mount `/docker-entrypoint-initdb.d` to the container:

- https://hub.docker.com/r/bitnami/openldap/#:~:text=Initializing%20a%20new%20instance

Reply:
in either case. remove `-e` from echo. not needed